### ISSUE TITLE
using term instead of @formula from FixedEffectModels causes an error

### ISSUE THREAD

**Issue by jjgecon** [jmboehm/RegressionTables.jl#142]
I'm running 

```
using RegressionTables, DataFrames, FixedEffectModels, RDatasets, GLM

df = dataset("datasets", "iris")

rr1 = reg(df, @formula(SepalLength ~ SepalWidth + fe(Species)))
rr2 = reg(df, term(:SepalLength) ~ term(:SepalWidth) + fe(Species)))
```
Running the ```regtable(rr1)``` yields the expected return however running

```
regtable(rr2)
```
yields `MethodError: no method matching name(::FixedEffectModels.FixedEffectTerm)`.

I suspect it has something to do with how the fixed effects are stored in the formula. With `@formula` they are stored as `(Species)->fe(Species)` while using terms are stored as `FixedEffectModels.FixedEffectTerm(:Species)`.

I was working a few months ago.

**Comment by junder873** [comment]
Thank you for the report, however, I am struggling to replicate it:
```julia
using RegressionTables, DataFramesMeta, FixedEffectModels, RDatasets, GLM

df = dataset("datasets", "iris")

rr1 = reg(df, @formula(SepalLength ~ SepalWidth + fe(Species)))
rr2 = reg(df, term(:SepalLength) ~ term(:SepalWidth) + fe(:Species))

regtable(rr2)

# -----------------------------------
#                         SepalLength
# -----------------------------------
# SepalWidth                 0.804***
#                             (0.106)
# -----------------------------------
# Species Fixed Effects           Yes
# -----------------------------------
# N                               150
# R2                            0.726
# Within-R2                     0.281
# -----------------------------------
```

Which version of RegressionTables are you using?

**Comment by jjgecon** [comment]
Thank you so much for the quick response.

I'm using RegressionTables v0.5.10. The package manager says there may be an upgrade, but it doesn't update.

I tried it on another computer with v0.5.9 and it works fine.

**Comment by junder873** [comment]
It looks like all that needs to be defined is a `name` method for the `FixedEffectTerm`, running:
```julia
RegressionTables.name(t::FixedEffectModels.FixedEffectTerm) = string(t.x)
```
before running `regtable` seems to fix it for me on v0.5.10.

**Comment by jjgecon** [comment]
This works for me. Thanks!
